emgapianns/models.py

```diff
@@ -321,7 +322,11 @@ class Organism(mongoengine.Document):
     pipeline_version = mongoengine.StringField(required=True)
 
     meta = {
-        'ordering': ['lineage']
+        'ordering': ['lineage'],
+        'auto_create_index': True,
```

Comment:
The first time we execute this we need to use the mongo admin user, otherwise the index creation will fail

Reply:
yeah we should include that somewhere in the documentation, where do you think should be the best place for it, confluence?